skbeam/io/save_powder_output.py

```diff
@@ -250,42 +254,3 @@ def _validate_input(tth, intensity, err, ext):
     if ext == '.xye' and err is None:
         raise ValueError("Provide the Error value of intensity"
                          " (for .xye file format err != None)")
-
-
-def _create_file_path(dir_path, output_name, ext):
-    """
```

Comment:
When debugging over gchat i may have suggested this function should be moved to fit2d_mask.py when it was in the __init__.py of this subpackage. That was wrong. I'd prefer to leave this function in this module is unless you've got a compelling reason to want it in fit2d_mask.py